jbpm-flow/src/main/java/org/jbpm/workflow/instance/node/RuleSetNodeInstance.java

```diff
@@ -119,7 +121,7 @@ public void internalTrigger(final NodeInstance from, String type) {
                 }
 
                 if (decision != null && !decision.isEmpty()) {
-                    dmnResult = runtime.evaluateDecisionByName(dmnModel, decision, context);
+                    dmnResult = runtime.evaluateByName(dmnModel, context, decision);
```

Comment:
@tarilabs is this right refactor to not use deprecated method?

Reply:
indeed as per Javadocs. Thanks!

Reply:
cool, thanks